### DISCUSSION TITLE
New space for discussing the UE4-SpaceMouse Plugin

### DISCUSSION THREAD

**Discussion by microdee** [microdee/OpenUnrealSpaceMouse#18] [General]
This is more like a forum than the github issues, so it can take over from the long and convoluted thread on the UE4 forums.

**Comment by Nekbeth** [comment]
Hopefully you can add support to Mac for next update  :)

**Reply by microdee** [reply]
I'm actually working on implementing the official 3DConnexion navigation library. When that's done I hope it'll just work on MacOS, because the raw HID communication didn't work out.
